modules/minileven/minileven.php

```diff
@@ -34,8 +34,6 @@ function jetpack_check_mobile() {
 		return false;
 	if ( jetpack_mobile_exclude() )
 		return false;
-	if ( 1 == Jetpack_Options::get_option_and_ensure_autoload( 'wp_mobile_disable', '0' ) )
```

Comment:
In addition to straight removing this, should we clean up after ourselves and delete `wp_mobile_disable`?